### ISSUE TITLE
Exception in native call from JS

### ISSUE THREAD

**Issue by Somnus007** [alan-ai/alan-sdk-reactnative#13]
Versions:
Device OS version: Android 9
React native version: 0.65.1
@alan-ai/alan-sdk-react-native: 1.6.0

Integrate code:

- Subscribe
```
import {NativeEventEmitter, NativeModules} from 'react-native';
const {AlanManager, AlanEventEmitter} = NativeModules;
const alanEventEmitter = new NativeEventEmitter(AlanEventEmitter);

const alanSubscriber = alanEventEmitter.addListener('command', data => {
  console.log(`got command event ${JSON.stringify(data)}`);
});
```

- Alan Button
`{alanKey && <AlanView projectid={alanKey} />}`

Reproduce steps:

1. Run app in development mode on Android device
2. Alan AI works well
2. Exit the application
3. Reopen the application
4. The error will occur
5. Remove `AlanView`, the error won't occur.

![screenshot](https://user-images.githubusercontent.com/31230649/132479231-226c018f-de2a-475f-aeae-e0f847815747.jpg)

**Comment by Somnus007** [comment]
Hi @snyuryev , any updates on this?

**Comment by snyuryev** [comment]
Hi @Somnus007 
1. Please try new version of our plugin (1.8.0)
2. If the problem still appears - could you share a full source code?

What is your host OS? Windows/Mac/.. ?

**Comment by Somnus007** [comment]
Versions:
Device OS version: Android 11
React native version: 0.66.1
@alan-ai/alan-sdk-react-native: 1.8.0
Development OS: Mac OS Monterey

1. I just created a new demo project via `npx react-native init AwesomeProject`
2. then installed Alan SDK via `yarn add @alan-ai/alan-sdk-react-native`
3. Only add below code in App.js
`import { AlanView } from '@alan-ai/alan-sdk-react-native';`
```
<AlanView
      projectid="my project id"
    />
```
4. The error occured again.

**Comment by Somnus007** [comment]
Hi @snyuryev , [here](https://github.com/Somnus007/alan-ai-demo) is my new demo project.
Please replace `YOUR PROJECT ID` with your own project id in `App.js`

**Comment by snyuryev** [comment]
Just tried this steps:
1. ```npx react-native init AwesomeProject```
2. ```cd AwesomeProject```
3. ```npm i @alan-ai/alan-sdk-react-native```
4. ```npx react-native run-android```
5. App is running ok. Alan button is working.
6. Close app.
7. Run app again.
8. App is running ok. Alan button is working.

Tried both emulator and real device. The only difference. I put  ```<AlanView projectid="my project id"/>``` after ```</ScrollView>``` in App.js. It is not required to put it in a scroll view. I guess that will fix your issue.

**Comment by Somnus007** [comment]
Hi @snyuryev , I got the error again even I put `<AlanView projectid="my project id"/>` after `</ScrollView>`. How did you run the app again? In my case, I just reopen app by clicking app icon on device, NOT run `npx react-native run-android` again.
Can you share your new created demo project code?

**Comment by snyuryev** [comment]
Yes, the same way as you are describing. Run once with ```npx react-native run-android```, kill the app and reopen from device (or emulator). App is loaded fine, without any issues. 

What if you run the app with Android Studio?

**Comment by Somnus007** [comment]
I got the same error when running app with Android Studio.
Cloud you please share your demo project code?

**Comment by snyuryev** [comment]
@Somnus007 Demo project is [here](https://github.com/snyuryev/AwesomeProject)

**Comment by Somnus007** [comment]
Just cloned your project, put my own project id,
- `cd AwesomeProject`
-  `npm install`
-  `npx react-natvie run-android`
-  run app, close app, reopen app, then error occured

https://user-images.githubusercontent.com/31230649/140308412-286f6bd6-d7ee-4c7f-b3f0-8b8ee2e5ed04.mp4

**Comment by Somnus007** [comment]
Hi @snyuryev , are there any other possible solutions that can help me get rid of this error?

**Comment by snyuryev** [comment]
Hi @Somnus007 
We are trying to reproduce this issue and tried different env. 
For now I can suggest reinstall react native cli, metro, remove node_modules, re-install deps and try to run app again.

**Comment by Somnus007** [comment]
Ok, yesterday I even tried to reinstall my mac OS and rebuilt my development environment, but the error still persists. I also tried to build a new react native application in my colleague's mac, and the same error occurred.

**Comment by snyuryev** [comment]
Hi @Somnus007 
We are trying to fix this issue. Could you provide some additional information. 
Do you have the same issue with release version of the app?
Do you see any logs in metro server then you open your app second time? Something like
```
 LOG  Running "AwesomeProject" with {"rootTag":1}
 BUNDLE  ./index.js 
 ```

**Comment by Somnus007** [comment]
Hi @snyuryev ,
For the released version of the application, I opened it for the first time after installation and it worked well. Then I closed it and reopened it again, the application crashed and exited. I reopened it(third time) and everything worked fine.
The second time I opened the app in the development environment, there was only one line of log: `LOG  Running "test" with {"rootTag":11}`

**Comment by snyuryev** [comment]
@Somnus007 I've updated [example](https://github.com/snyuryev/AwesomeProject) - added node_modules folder with slightly updated AlanSDK plugin. Could you please try it.

**Comment by Somnus007** [comment]
Hi @snyuryev ,

- Using your example
        1. Turn off `Wake Word In Background` on Alan studio
        2. `npx reacr-native run-android`, everything goes well
        3. Close app and reopen it, no crash occured.
  
- Using a new demo project with current `@alan-ai/alan-sdk-react-native 1.8.0`
        1. Turn off `Wake Word In Background` on Alan studio
        2. `npx reacr-native run-android`, everything goes well
        3. Close app and reopen it, no crash occured.

Conclusion: 
1. It seems that this bug is caused by the `Wake Word In Background` option on Alan Studio
2. We need the `Wake Word In Background` function, could you please fix this bug with the "Wake Word In Background" option turned on?
3. In your example, I can toggle showing the Alan Button by clicking the `SHOW/HIDE BUTTON`. It looks good, but I also noticed that even if the Alan Button is hidden, I can still wake up Alan via `Hey Alan`. So I don't think it was really removed, it just became transparent. Is there a way to make the Alan Button really be removed and stop working?

**Comment by snyuryev** [comment]
Fixed with version 1.10.0
